pr_testing/run-pr-relvals.sh

```diff
@@ -79,27 +79,30 @@ popd
 TEST_ERRORS=$(grep -ai -E "ERROR .*" ${LOG} | grep -v 'DAS QL ERROR' | grep -v 'ERROR failed to parse X509 proxy') || true
 GENERAL_ERRORS=`grep -a "ALL_OK" ${LOG}` || true
 
+RESULT_FILE_NAME=$(get_result_file_name relval ${TEST_FLAVOR} results)
+FAILED_FILE_NAME=$(get_result_file_name relval ${TEST_FLAVOR} failed)
+
 if [ "X$TEST_ERRORS" != "X" -o "X$GENERAL_ERRORS" = "X" ]; then
   echo "Errors in the RelVals"
-  echo "MATRIX${UC_TEST_FLAVOR}_TESTS;ERROR,Matrix ${UC_TEST_FLAVOR} Tests Outputs,See Logs,runTheMatrix${UC_TEST_FLAVOR}-results" >> ${RESULTS_DIR}/relval${UC_TEST_FLAVOR}.txt
+  echo "MATRIX${UC_TEST_FLAVOR}_TESTS;ERROR,Matrix ${UC_TEST_FLAVOR} Tests Outputs,See Logs,runTheMatrix${UC_TEST_FLAVOR}-results" >> $(get_status_file_name relval $TEST_FLAVOR)
```

Comment:
@iarspider , looks like you are missing `${RESULTS_DIR}/`  for these status/resuots files